### DISCUSSION TITLE
File system analytics - understanding volume_analytics_bytes_used_by_accessed_time

### DISCUSSION THREAD

**Discussion by cjproud** [NetApp/harvest#2746] [Q&A]
Hi there,

I'm trying to understand what's returned from the volume_analytics_bytes_used_by_accessed_time metric. I can see that for a particular directory (dataset-1) we get multiple metrics back for that directory with different access times and a different "order" parameter:

![Screenshot 2024-03-13 at 10 12 15 am](https://github.com/NetApp/harvest/assets/41252312/5b11e2f9-d2b3-4413-a9f6-9b1775cbf2b4)

What does that order parameter mean and how does it relate to the access time? Is it used as a sorting index on the access time?

**Comment by rahulguptajss** [comment]
@cjproud You are correct. The `order` is used to sort below panels in the FSA dashboard. Lower the `order` label, the more recent the record is for that relevant directory and activity(Weekly/Monthly/Quarterly/Yearly).

<img width="1850" alt="image" src="https://github.com/NetApp/harvest/assets/25551691/6976084a-c1d3-469e-bfba-85511551711f">

**Reply by cjproud** [reply]
Great, thanks for that @rahulguptajss! So if I wanted to find what the most recent access time is for that directory, I should just grab the lowest order value and the corresponding access time?

I realise I made a small mistake in my example, those access times are from the same dataset - like so:

<img width="2109" alt="Screenshot 2024-03-14 at 4 34 18 pm" src="https://github.com/NetApp/harvest/assets/41252312/28899190-621c-45c7-9b4b-8b4745a8d19a">

Why is it that the Data used column is sometimes 0 bytes? Is that because some sub-directories within that directory contain 0 bytes?

**Reply by rahulguptajss** [reply]
@cjproud That is correct. You also need to take the `activity` label into account. So, if you select the lowest order value for an activity (Weekly/Monthly/Quarterly/Yearly) and directory, you should be able to retrieve the latest accessed data for that activity and directory.

The `Data Used` column in your screenshot indicates that in the month of March 2024, for the dataset-1 directory, 514 MB of data was accessed. In contrast, for the month of February 2024, no data (0B) was accessed from this directory.
